internal/builders/docker/commands.go

```diff
@@ -53,14 +56,14 @@ func DryRunCmd(check func(error)) *cobra.Command {
 	return cmd
 }
 
-func writeBuildDefinitionToFile(bd pkg.BuildDefinition, path string) error {
-	bytes, err := json.Marshal(bd)
+func writeToFile[T any](obj T, path string) error {
+	bytes, err := json.Marshal(obj)
 	if err != nil {
-		return fmt.Errorf("couldn't marshal the BuildDefinition: %v", err)
+		return fmt.Errorf("marshaling the object failed: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("marshaling the object failed: %w", err)
```

Reply:
Done. Is this the convention everywhere, or only for errors that need to be unwrapped?